tests/e2e/utils/shared.ts

```diff
@@ -75,6 +75,20 @@ export const skipCtbTour = async (page: Page) => {
   }
 };
 
+/**
+ * Clicks on a link and waits for the page to load completely.
+ *
+ * NOTE: this util is used to avoid inconsistent behaviour on webkit
+ *
+ */
+export const clickLocatorAndWaitForLoad = async (page: Page, locator: Locator) => {
```

Comment:
Maybe just "clickAndWait" or "safeClick" or something shorter and more generic? We'll use it almost everywhere and the important part is that it's a click we can rely on